air/src/options.rs

```diff
@@ -63,7 +216,7 @@ pub struct ProofOptions {
     grinding_factor: u8,
     field_extension: FieldExtension,
     fri_folding_factor: u8,
-    fri_max_remainder_size: u8, // stored as power of 2
+    fri_max_remainder_size: u32,
```

Comment:
any particular reason why this is increased? (I'm not against, but as it is orthogonal to the refactoring, I'm asking)

Reply:
@Nashtare In fact, it looks like we were previously getting lucky here.  This field is currently inhabited by the result of calling `trailing_zeros` on a primitive integer type.  That call always returns a `u32`, and that means that there are a lot of values that call could return that are outside the bounds of what is representable as a `u8`.

Reply:
Ah indeed, good point.